packages/e2e-test-utils/src/uploadMedia.js

```diff
@@ -27,7 +27,7 @@ import uploadFile from './uploadFile';
  */
 async function uploadMedia(file, exit = true) {
   // Clicking will only act on the first element.
-  await expect(page).toClick('button', { text: 'Upload' });
+  await expect(page).toClick('button[aria-label="Upload"]');
```

Comment:
Is this related to this changed?

Reply:
Previously, there was a button with text `Upload`, now the button is an icon but still has a label `Upload`, so the selector had to change a bit. 
I just added this change since E2E tests were failing before.